astroquery/mast/cutouts.py

```diff
@@ -637,3 +637,163 @@ def get_cutouts(self, coordinates, *, size=5, survey=None):
+
+        super().__init__()
+
+        services = {"astrocut": {"path": "astrocut"}}
+
+        self._service_api_connection.set_service_params(services, "hapcut")
+
+    def download_cutouts(self, coordinates, *, size=5, path=".", inflate=True):
+        """
+        Download cutout images around the given coordinates with indicated size.
+
+        Parameters
+        ----------
+        coordinates : str or `astropy.coordinates` object
+            The target around which to search. It may be specified as a
+            string or as the appropriate `astropy.coordinates` object.
+        size : int, array-like, `~astropy.units.Quantity`
+            Optional, default 5 pixels.
+            The size of the cutout array. If ``size`` is a scalar number or
+            a scalar `~astropy.units.Quantity`, then a square cutout of ``size``
+            will be created.  If ``size`` has two elements, they should be in
+            ``(ny, nx)`` order.  Scalar numbers in ``size`` are assumed to be in
+            units of pixels. `~astropy.units.Quantity` objects must be in pixel or
+            angular units.
+        path : str
+            Optional.
+            The directory in which the cutouts will be saved.
+            Defaults to current directory.
+        inflate : bool
+            Optional, default True.
+            Cutout target pixel files are returned from the server in a zip file,
+            by default they will be inflated and the zip will be removed.
+            Set inflate to false to stop before the inflate step.
+
+        Returns
+        -------
+        response : `~astropy.table.Table`
+        """
+
+        # Get Skycoord object for coordinates/object
+        coordinates = parse_input_location(coordinates)
+
+        # Build initial astrocut request
+        astrocut_request = f"astrocut?ra={coordinates.ra.deg}&dec={coordinates.dec.deg}"
+
+        # Add size parameters to request
+        size_dict = _parse_cutout_size(size)
+        astrocut_request += f"&x={size_dict['x']}&y={size_dict['y']}&units={size_dict['units']}"
+
+        # Build the URL
+        astrocut_url = self._service_api_connection.REQUEST_URL + astrocut_request
+
+        # Set up the download path
+        path = os.path.join(path, '')
+        zipfile_path = "{}hapcut_{}.zip".format(path, time.strftime("%Y%m%d%H%M%S"))
+
+        # Download
+        self._download_file(astrocut_url, zipfile_path)
+        localpath_table = Table(names=["Local Path"], dtype=[str])
+
+        # Checking if we got a zip file or a json no results message
+        if not zipfile.is_zipfile(zipfile_path):
+            with open(zipfile_path, 'r') as FLE:
+                response = json.load(FLE)
+            warnings.warn(response['msg'], NoResultsWarning)
+            return localpath_table
+
+        if not inflate:  # not unzipping
+            localpath_table['Local Path'] = [zipfile_path]
+            return localpath_table
+
+        print("Inflating...")
```

Comment:
Don't use prints, us the logger or a verbose keyword instead. This should be changed in the other classes, too.